Makefile

```diff
@@ -115,7 +115,9 @@ else
 endif
 	# This might help with emfile errors during `make javascript`: ulimit -n 10240
 	@rm -rf dev/lib
-	@dev/run -n 1 -q --with-admin-party-please test/javascript/run $(suites)
+	@dev/run -n 1 -q --with-admin-party-please \
+            -c 'startup_jitter=0' \
```

Comment:
I'm +1 on the entire concept, and -0 on the implementation. For instance, if this grows beyond 1-2 settings we should probably implement it differently.

Recently I added the `local.d` and `default.d` directories to the overlay folders. Perhaps instead you should just create a file `local.d/startup-jitter.ini` and put your setting in there. In fact, a lot of the machinations `dev/run` goes through are obsolete if we just move to creating `local.d/*.ini` files instead of in-place substitutions.

Don't let this block getting your PRs landed, but if you don't tackle it, would you file an issue please? and I might look at it sooner rather than later.